garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stools/axis.lua

```diff
@@ -147,12 +148,10 @@ function TOOL:RightClick( trace )
 
 		-- Note: To keep stuff ragdoll friendly try to treat things as physics objects rather than entities
 		local Ang1, Ang2 = Norm1:Angle(), ( -Norm2 ):Angle()

```

Comment:
```suggestion
		--local Ang1, Ang2 = Norm1:Angle(), ( -Norm2 ):Angle()

```
this is what you wanted to comment out